charts/pravega-operator/README.md

```diff
@@ -4,16 +4,17 @@ Installs [pravega-operator](https://github.com/pravega/pravega-operator) to crea
 
 ## Introduction
 
-This chart bootstraps a [pravega-operator](https://github.com/pravega/pravega-operator) deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager. The chart can be installed multiple times to create Pravega Operator on multiple namespaces.
+This chart bootstraps a [pravega-operator](https://github.com/pravega/pravega-operator) deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
```

Comment:
I don't see why we need a document to explain the operator _helm chart._
This should really be a document titled **Deploying pravega-operator**

Reply:
Are you asking to remove the Introduction section all together?

Reply:
Yes the Introduction section can be removed and this document should be titled "Deploying Pravega-Operator", subsections can be pre-requisites, install, configuration, uninstall and upgrade.